Jenkinsfile

```diff
@@ -55,7 +55,7 @@ pipeline {
                     def stanMathSigs = ['test/integration/signatures/stan_math_sigs.expected'].join(" ")
                     skipExpressionTests = utils.verifyChanges(stanMathSigs)
 
-                    def sourceCodePaths = ['src'].join(" ")
+                    def sourceCodePaths = ['src', 'test/integration/good', 'test/stancjs'].join(" ")
```

Comment:
We should run all tests also when we just add new tests.

Reply:
I agree, though ideally we wouldn't rebuild all the binaries if only `test/` changed

Reply:
Good call. Done.